### PR TITLE
Capstone project

### PR DESCRIPTION
**Pull request by Isabellanamutenda**
In this project I:
- Created a repository named capstone for this project
- Created a branch for all the work of this project
- Added the .github/workflows/linters.yml
- Added gitignore file
- Added .hintrc file
- Added stylelintrc file
- Added estlintrc file
- Update the README file
- Added index.html and about.html file
- Added js files
- Added a live demo

-

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,55 @@
...
+ - Add it and commit it
+ - Set up the linters of javascript, html and css
+ - Add it ad commit
+ - Install the modules and packages
+ - Complete the project by coding
+
+
+### Run tests
+-This is just to make sure that all the work is running perfectly and with no errors.
+
```
Comment (bellom) [inline README.md:41]: Let's credit the author in our README.

image

File: README.md [hunk 2]
```diff
@@ -1 +1,55 @@
...
+> This is the capstone projecr for html, css, and javascript DOMmaniputation of the first week.
+
+
+## Built With
+
+- HTML
+- CSS
+- Javascript
+
+### Live demo
```
Comment (bellom) [inline README.md:12]: Oops, you're missing the live demo link.

Kindly make use of GitHub Pages to deploy the web pages and add the live demo link.

File: about.html [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+        <img src="img/partner5.png" class="partners" alt="Partner logo">
+      </div>
+    </section>
+
+    <footer>
+      <div class="footer-container">
+        <img class="footer-logo" src="img/cc-logo.png" alt="Company logo">
+        <p class="footer-text"> 2015 Creative Commons Korea <br> Some Rights Reserved.<br></p>
+      </div>
+    </footer>
```
Comment (bellom) [inline about.html:78]: Let's work on the footer for about page.

Mobile:
White footer 
image


Desktop:
Black footer
image

File: style.css [hunk 4]
```diff
@@ -0,0 +1,698 @@
...
+  margin: 0;
+}
+
+.bold {
+  font-weight: bold !important;
+  font-family: cocogoose-b, sans-serif;
+  font-size: 1.1em;
+}
+
+#menu-top {
```
Comment (bellom) [inline style.css:65]: ### [Optional]
Your code appears to be WET-(Write Every Time), Microverse encourage DRY-(Don't Repeat Yourself) code.

For example In your case; you repeated `display: flex;`   `flex-direction: column;`, and so on...

Could be combined like this:

#menu-top,  #welcome.....{
  display: flex;
  flex-direction: column;
}

Let's improve our styling by using [Combine Elements](https://speckyboy.com/good-bad-css-practices)

**Review by bellom — CHANGES_REQUESTED** [review]
## Hi @Gunjuzone  👏 , 

### Awesome job so far!! :+1: 

#### You have done wonderfully well setting up linter and good commit messages 🔥, However, you passed `almost` all the checklist for this project :+1: just a few issues left to work on............ :slightly_smiling_face: 

**General:**
Your code looks great, is well-organized, and also you have an awesome responsive design 👍.

# Status: Required Changes! ♻️ 


**Video Presentation:**

We are missing the video link.

image

Kindly check my comment under the review.


### ♻️ Kindly re-submit for review after making all those changes :slightly_smiling_face: Thumps -up await you.....

### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 2 above)
Comment (Isabellanamutenda) [reply README.md:12]: Resolved

File: README.md (hunk 1 above)
Comment (Isabellanamutenda) [reply README.md:41]: Resolved

File: about.html (hunk 3 above)
Comment (Isabellanamutenda) [reply about.html:78]: Resolved

File: style.css (hunk 4 above)
Comment (Isabellanamutenda) [reply style.css:65]: I will work on it and improve that 
Thank you for that

File: README.md (hunk 1 above)
Comment (Lameck1) [reply README.md:41]: Resolved 👍🏼

File: README.md (hunk 2 above)
Comment (Lameck1) [reply README.md:12]: Resolved 👍🏼

File: about.html (hunk 3 above)
Comment (Lameck1) [reply about.html:78]: Resolved 👍🏼

**Review by Lameck1 — APPROVED** [review]
Hi, @Isabellanamutenda,

I've verified that you took note and addressed all concerns from the previous review. Your project is now complete!  It's time to merge it :shipit: 

I highly encourage that you consider the optional suggestion too in this and your next projects. All the best.

Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**